src/views/LobbyView.vue

```diff
@@ -74,6 +85,14 @@
       </v-col>
       <v-spacer />
     </v-row>
+    <BaseSnackbar
+      v-model="gameStore.showIsRankedChangedAlert"
+      :timeout="2000"
```

Comment:
We haven't defined a timeout prop for `BaseSnackbar` so I believe this won't do anything. `<v-snackbar>` defaults to 5s timeout so I think we can just remove this and reasonably use the default atm
```suggestion
```